guide/C/ch_currency.xml

```diff
@@ -1353,15 +1342,15 @@ Trading
 
         <para>At this point there have been no trading gains or losses. The balances of the USD and HKD trading
           accounts are the same when converted to USD. The HKD trading account has a debit (-ve
-          balance) and the USD trading account has a credit balance (+ve balance). The nett balance
+          balance) and the USD trading account has a credit balance (+ve balance). The net balance
```

Comment:
What ve means?

vacuum extraction, vacuum extractor, value engineering, ventilating equipment, vernal equinox, 
vernier engine, visual efficiency, vitamin E, Venezuela, final military Victory in Europe

Reply:
I think it's supposed to be positive and negative. @DaveC49, is that correct?

Reply:
No response from @DaveC49 so please change +ve to positive and -ve to negative.

Reply:
79988cac